src/compounds/Messages/Messages.jsx

```diff
@@ -1,25 +1,88 @@
-import React from 'react'
-import { Card } from 'react-bootstrap'
+import React, { useState } from 'react'
 import PropTypes from 'prop-types'
+import { Card, Collapse } from 'react-bootstrap'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { timeSince, formatBytes } from '../../resources/utilityFunctions'
+import Link from '../../components/Link/Link'
 import Title from '../../components/Title/Title'
 import TextBox from '../../components/TextBox/TextBox'
 
-const Messages = ({ addClass, messages }) => (
+const MessageCardBody = ({messages, message, idx}) => {
+  const { avatar, body, id, name, attachments, dateAndTime } = message
+  return (
+    <Card key={id} className={`${idx === (messages.length - 1) ? '' : 'pb-4'} pt-4 flex-row border-0 border-top`}>
+      <Card.Img variant='left' src={avatar} className='h-25' />
+      <Card.Body className='pt-0 pe-0 d-flex flex-column flex-md-row'>
+        <div>
+          <Card.Title>{name}</Card.Title>
+          <Card.Text>
+            {body}
+          </Card.Text>
+          {(attachments.length > 0) && (attachments.map((attachment) => {
+              const { content_length, filename, uuid } = attachment 
+              // this href in the link below is the URLto download attachments from the marketplace- this works, since all these attachments already show up in the marketplace, and these requests will always be in the marketplace as well as the webstore
+              // but, not sure if we should be using the URL from the marketplace or creating a new, webstore based url from the "download" property. download property returns a string like this example: "/quote_groups/10bd825f-c7e0-4c07-bd9e-f7982b21f435/attachments/nepal5.png"
```

Comment:
@alishaevn let me know your thoughts on this.. I can't think of an instance where this link would not work, since the user will need to be signed in with oauth & on this page to have access to this link regardless.  if I'm signed out from the marketplace, I cannot use these links to download the document. but, its also its possible this is not the right way to do it